ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -261,7 +261,16 @@ export default class ExperimentalTab extends PureComponent {
               id="select-theme"
               options={themesOptions}
               selectedOption={theme}
-              onChange={async (newTheme) => setTheme(newTheme)}
+              onChange={(newTheme) => {
+                this.context.trackEvent({
+                  category: 'Settings',
+                  event: 'Theme Changed',
+                  properties: {
+                    theme_selected: newTheme,
+                  },
+                });
+                setTheme(newTheme);
+              }}
```

Comment:
I suspect an inline function here can cause re-rendering in `<Dropdown` `useCallback` may be a good idea here.